deploy/builder/templates-install-bundle/clickhouse-operator-install-yaml-template-01-section-crd-01-chi-chit.yaml

```diff
@@ -308,6 +313,13 @@ spec:
                   enum:
                     - ""
                     - "RollingUpdate"
+                suspend: &TypeStringBool
```

Comment:
Syntax is incorrect still, good example is `troubleshoot` option

Reply:
![Screenshot 2024-12-09 at 10 28 27 AM](https://github.com/user-attachments/assets/e74040d4-c152-4f53-bca0-1005e4969ce5)